README.md

```diff
@@ -31,7 +31,7 @@ I'd like to take a few sentences to clarify my stance on `gofmt` because there a
       * [Returning Defined Errors](#Returning-Defined-Errors)
       * [Returning Dynamic Errors](#Returning-Dynamic-Errors)
     * [Pointers in Go](#Pointers-in-Go)
-    * [Closures are Function Pointers](#Closures-are-Function-Pointers)
+    * [Closures Are Function Pointers](#Closures-are-Function-Pointers)
```

Comment:
Nope, since "are" is a verb, it must be capitalized when appearing in a title :) There's a rule of thumb that you should lowercase anything that's four (three? I can't remember) letters or shorter, but it fails here.